pkg/converter/convert_test.go

```diff
@@ -11,6 +12,7 @@ import (
 )
 
 func TestConvert(t *testing.T) {
+	os.Setenv("KUBECONFIG", "/tmp/foo.conf")
```

Comment:
1. we should not hard code it as it's OS dependent. there is function to create temp directory.
2. we should remove this directory at the end of the test (defer)